prime-router/metadata/schemas/covid-19.schema

```diff
@@ -800,6 +800,9 @@ elements:
     natFlatFileField: Testing_lab_id
     hhsGuidanceField:
     hl7Field: #OBX-23-10
+    documentation: |
+        Currently set to default to the CLIA ID for Via Elegante, Tucson Mountains. If another lab sends an empty ID,
+        it will look like it came from Tucson mountains
```

Comment:
FYI, I actually fixed this hack, so this particular comment isn't really true any more.  Dunno if you'll see a conflict on merge.